src/components/NewCommentForm.tsx

```diff
@@ -1,37 +1,104 @@
-import React from 'react';
+import React, { FormEvent, useState } from 'react';
+import classNames from 'classnames';
+import { CommentData } from '../types/Comment';
+
+type Props = {
+  onAddComment: (commnet: CommentData) => Promise<void>;
+};
+
+export const NewCommentForm: React.FC<Props> = ({ onAddComment }) => {
+  const [nameInput, setNameInput] = useState('');
+  const [emailInput, setEmailInput] = useState('');
+  const [commentInput, setCommentInput] = useState('');
+  const [nameError, setNameError] = useState(false);
+  const [emailError, setEmailError] = useState(false);
+  const [commentError, setCommentError] = useState(false);
```

Comment:
Same here

Reply:
https://github.com/mate-academy/react_dynamic-list-of-posts/pull/873#discussion_r1281792440

didn't fix it

Reply:
This issue is still not fixed
Let's create one state for errors and put here an object for all error options
for instance:

```javascript
const [inputErrors, setInputErrors] = useState({
  nameError: '',
  emailError: '',
  commentError: '',
})

```